src/components/Button/styles.ts

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+
+export const ButtonReset = styled.button`
+  background-color: white;
+  color: #c94545;
+  font-family: "Nunito", sans-serif;
+  font-weight: 600;
+  border-radius: 3px;
+  border: none;
+  padding: 0.5rem 1rem 0.5rem 1rem;
+  cursor: pointer;
+`;
+export const ButtonDefault = styled.button`
+  font-size: 1.5rem;
+  font-weight: 600;
+  font-family: "Nunito", sans-serif;
+  display: flex;
```

Comment:
[suggestion] You also don't need the buttons to be flex, so you can remove this line.